project/OcsBundle.scala

```diff
@@ -242,6 +242,7 @@ trait OcsBundle {
       bundle_edu_gemini_spModel_core,
       bundle_edu_gemini_spModel_io,
       bundle_edu_gemini_spModel_pio,
+      bundle_edu_gemini_spModel_smartgcal,
```

Comment:
We need to make sure this runs on 2.12. Could we add that bundle to the travis check for 2.12?